source/Tests/UsingDeclarationAnalyzerFixture.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using NUnit.Framework;
+using Octopus.RoslynAnalyzers;
+using Verify = Microsoft.CodeAnalysis.CSharp.Testing.NUnit.AnalyzerVerifier<Octopus.RoslynAnalyzers.UsingDeclarationAnalyzer>;
+
+namespace Tests
+{
+    public class UsingDeclarationAnalyzerFixture
+    {
+        [Test]
+        public async Task DetectsUsingDeclaration()
+        {
+            var result = new DiagnosticResult(UsingDeclarationAnalyzer.Rule)
+                .WithSpan("", 10, 13, 10 ,42);
+
+            await Verify.VerifyAnalyzerAsync(UsingDeclaration, result);
+        }
+
+        [Test]
+        public async Task IgnoresUsingBlockWithBraces()
+        {
+            await Verify.VerifyAnalyzerAsync(UsingBlockWithBraces);
```

Comment:
How are these doing their job if we aren't supplying the "expected" params?

Reply:
The expected results is an array, so zero diagnostic results = no detections, that's how existing tests are doing it